src/hooks/useHandleTransfer.tsx

```diff
@@ -283,58 +308,250 @@ async function evm(
+            amountNormalizeAmount,
+            recipientChain,
+            zeroPad(arrayify(readableTargetAddress!), 32),
+            THRESHOLD_ARBITER_FEE,
+            THRESHOLD_NONCE
+          );
+
+          // We increase the gas limit estimation here by a factor of 10% to account for
+          // some faulty public JSON-RPC endpoints.
+          const gasLimit = estimateGas.mul(1100).div(1000);
+          const overrides = {
+            gasLimit,
+            // We use the legacy tx envelope here to avoid triggering gas price autodetection using EIP1559 for polygon.
+            // EIP1559 is not actually implemented in polygon. The node is only API compatible but this breaks some clients
+            // like ethers when choosing fees automatically.
+            ...(chainId === CHAIN_ID_POLYGON && { type: 0 }),
+          };
+
+          console.log("sendTbtc overrides", { overrides });
+
+          const tx = await L2WormholeGateway.sendTbtc(
+            amountNormalizeAmount,
+            recipientChain,
+            zeroPad(arrayify(readableTargetAddress!), 32),
+            THRESHOLD_ARBITER_FEE,
+            THRESHOLD_NONCE,
+            overrides
+          );
+
+          const receipt = await tx.wait();
+          console.log({ receipt });
+
+          dispatch(
+            setTransferTx({
+              id: receipt.transactionHash,
+              block: receipt.blockNumber,
+            })
+          );
+
+          enqueueSnackbar(null, {
+            content: <Alert severity="success">Transaction confirmed</Alert>,
+          });
+
+          const sequence = parseSequenceFromLogEth(
+            receipt,
+            getBridgeAddressForChain(chainId)
+          );
+          const emitterAddress = getEmitterAddressEth(
+            getTokenBridgeAddressForChain(chainId)
+          );
+
+          console.log({ sequence, emitterAddress });
+
+          await fetchSignedVAA(
+            chainId,
+            emitterAddress,
+            sequence,
+            enqueueSnackbar,
+            dispatch
+          );
+
+          // const { vaaBytes, isPending } = await getSignedVAAWithRetry(
+          //   chainId,
+          //   emitterAddress,
+          //   sequence
+          // );
+
+          // if (vaaBytes !== undefined) {
+          //   console.log("uint8ArrayToHex(vaaBytes)", uint8ArrayToHex(vaaBytes));
+          //   const signedVAA = uint8ArrayToHex(vaaBytes);
+          //   console.log({ isPending });
+          //   console.log("Fetched Signed VAA");
+          // } else if (isPending) {
+          //   console.log({ isPending });
+          //   console.log("VAA is Pending");
+          // } else {
+          //   throw new Error("Error retrieving VAA info");
+          // }
+        } catch (e) {
+          console.error(parseError(e));
+        }
+      }
+    } else {
+      console.log("TRANSACCION FLUJO NORMAL PASANDO AQUI");
```

Comment:
console log in spanish, probably unecesary

Reply:
deleted ✅